### DISCUSSION TITLE
can Mixin Plugin shouldApplyMixin be used with connector?

### DISCUSSION THREAD

**Discussion by sleepingdragoninn** [Sinytra/Connector#53] [Q&A]
I've attempted to prevent forge-incompatible mixins with [this simple plugin](https://github.com/sisby-folk/tinkerers-smithing/blob/1.20/src/main/java/folk/sisby/tinkerers_smithing/mixin/ConnectorPlugin.java), however, even though `shouldApply` runs, the targets still attempt to resolve and the whole thing crashes. 

Are plugins supposed to work properly for this use case?

Here's a [log](https://mclo.gs/ZUy3imt) where i output mixins and their return values.

**Comment by sleepingdragoninn** [comment]
Oh, foolish, this is fixed in latest lol

**Reply by Su5eD** [reply]
Interesting, I wonder what the cause was. Well, hopefully it won't come back
